benchmarks/lockhammer/scripts/test_lockhammer.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import sys
+import os
+import sh
+import platform
+import unittest
+
+ALL_TESTS = []
```

Comment:
This does not have to be global does it?

Reply:
Yes, I can modify find_lh_test() and return a list instead of void